data/channels.csv

```diff
@@ -2486,7 +2490,7 @@ BBCNewsArabic.uk,BBC News Arabic,بي بي سي نيوز عربي,,BBC Studios,U
 BBCNordic.uk,BBC Nordic,,,BBC Studios,UK,,,general;entertainment,FALSE,2023-04-17,,,https://www.bbcnordic.com/,https://i.imgur.com/pC4pZuz.jpg
 BBCOne.uk,BBC One,BBC1;BBC Television;BBC Television Service,BBC Television,BBC,UK,,London,general,FALSE,1936-11-02,,,http://www.bbc.co.uk/bbcone/,https://upload.wikimedia.org/wikipedia/commons/thumb/8/8b/BBC_One_logo_2021.svg/512px-BBC_One_logo_2021.svg.png
 BBCParliament.uk,BBC Parliament,,,BBC,UK,,London,legislative,FALSE,1998-09-23,,,https://www.bbc.co.uk/tv/bbcparliament,https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/BBC_Parliament_Logo.svg/512px-BBC_Parliament_Logo.svg.png
-BBCPersian.uk,BBC Persian,تلویزیون فارسی بی‌بی‌سی,,BBC,UK,,London,news,FALSE,2009-01-14,,,https://www.bbc.com/persian,https://i.imgur.com/4uTMnPb.png
+BBCPersian.ir,BBC Persian,تلویزیون فارسی بی‌بی‌سی,,BBC,UK,,London,news,FALSE,2009-01-14,,,https://www.bbc.com/persian,https://i.imgur.com/4uTMnPb.png

```

Comment:
According to the article, the channel is broadcast from UK:

https://en.wikipedia.org/wiki/BBC_Persian